libraries/console_helpers.php

```diff
@@ -31,3 +31,22 @@ function info($result, $emptyLinesCount = 2)
 {
     echo $result . str_repeat(PHP_EOL, $emptyLinesCount);
 }
+
+function info_box(...$lines)
+{
+    foreach ($lines as $maxLines) {
+        $h1 = strlen($maxLines);
+        $lineValue[] = $h1;
+    }
+
+    $length = max($lineValue) + 6;
+
+    info (str_pad('*', $length, '*'), 1);
```

Comment:
```suggestion
    info(str_repeat('*', $length), 1);
```

Reply:
let's also move it to the separate console helper `write_symbol_line($length, $sybmol)`